shub/config.py

```diff
@@ -350,12 +350,18 @@ def get_image(self, target):
                 "Could not find image for project '{}'. Please define it "
                 "in your scrapinghub.yml.".format(target))
         elif image is False:
```

Comment:
`image` is `None` when you haven't configured image deploy, e.g. for
```
project: 123
```

`image` is `False` when the user explicitly disabled image deploy, e.g. for
```
project: 123
image: false
```
I think Viktor separated `None` and `False` to give a more helpful error message to the user on why their command failed.